src/TranslationFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend;
+
+use Symfony\Component\Translation\Loader\JsonFileLoader;
+use Symfony\Component\Translation\Translator;
+use Symfony\Component\Translation\MessageSelector;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class TranslationFactory {
```

Comment:
What motivated you to create this factory? (Wondering the same about `TwigFactory`)